node/Cargo.toml

```diff
@@ -39,10 +39,11 @@ sp-inherents = '2.0.0-alpha.6'
 sp-io = '2.0.0-alpha.6'
 sp-runtime = '2.0.0-alpha.6'
 grandpa-primitives = { package = 'sp-finality-grandpa', version = '2.0.0-alpha.6' }
+frame-benchmarking-cli = { version = "2.0.0-alpha.6" }
 
 # crust runtime modules
-crust-runtime = { path = '../runtime', version = '0.3.0' }
-cstrml-staking = { path = '../cstrml/staking', version = '0.3.0' }
+crust-runtime = { path = '../runtime', version = '0.3.0', features = ["std", "runtime-benchmarks"] }
```

Comment:
is it possible to make this toml simliar with staking/cargo.toml? I mean add `features` and the default feature is `std`